webpack-common.js

```diff
@@ -174,7 +174,7 @@ export function getPlugins({
       // This allow us to exclude locales for other apps being built.
       new webpack.ContextReplacementPlugin(
         /locale$/,
-        new RegExp(`^\\.\\/.*?\\/${appName}\\.js$`),
+        new RegExp(`^\\.\\/.*?\\/${APP_NAME}\\.js$`),
```

Comment:
I have the feeling that this isn't needed anymore.

Reply:
I thought that as well, so I removed it when working on the patch, but then the `bundlesize` test started failing, so this is evidently doing something to reduce the size of the bundle, but I'm not sure what. Maybe I have some leftover disco files on disk? I'll take another look.

Reply:
I changed this to be hardcoded as `amo`, and that works fine. I tried again to remove this logic and again the `bundlesize` test failed, so I am leaving it in for now.